imports/plugins/included/actionableAnalytics/client/components/Table/InventoryTable.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import _ from "lodash";
+
+export const InventoryTable = ({ fetchedTableData, tableHeadingArray }) => {
+  const tableHeading = tableHeadingArray.map(item =>
```

Comment:
This can be written as

```js
const tableHeading = tableHeadingArray.map(item => <th key={item}>{item}</th>);
```
There is no need for the parenthesis as the map callback only returns a single line